libafl_frida/src/helper.rs

```diff
@@ -471,6 +476,42 @@ impl<'a> FridaInstrumentationHelper<'a> {
         Aarch64Register::from_u32(regint as u32).unwrap()
     }
 
+    fn generate_maybe_log_blob(&mut self) -> Box<[u8]> {
+        macro_rules! maybe_log{
+            ($ops:ident) => {dynasm!($ops
+                ;   .arch x64
```

Comment:
We done my really need the macro, do we?

Reply:
No 😉 
I plan to move this to CoverageRuntime.